andela/Ah-frontend-guardians
# #162685616 Setup an empty react project

#### What does this PR do?
This PR sets up an empty react project.

#### Description of Task to be completed?
* Setup webpack
* Setup Babel

#### How should this be manually tested?
run the following commands
* git clone https://github.com/andela/Ah-frontend-guardians.git
* git checkout ch-empty-react-project-162685616
* npm install (to install all the dependencies in the package.json)
* npm start

Visit http://localhost:8080/
> The port:8080 may differ

you should see this
![image](https://user-images.githubusercontent.com/26769886/52557378-4a200180-2e00-11e9-9c2c-de6622e01fea.png)

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162685616


#### References:
I used this to setup the react project
https://www.robinwieruch.de/minimal-react-webpack-babel-setup/
